mod-source-record-manager-server/src/main/java/org/folio/services/AbstractChunkProcessingService.java

```diff
@@ -12,13 +12,7 @@
 import org.folio.dao.JobExecutionSourceChunkDao;
 import org.folio.dataimport.util.OkapiConnectionParams;
 import org.folio.kafka.exception.DuplicateEventException;
-import org.folio.rest.jaxrs.model.EntityType;
-import org.folio.rest.jaxrs.model.InitialRecord;
-import org.folio.rest.jaxrs.model.JobExecution;
-import org.folio.rest.jaxrs.model.JobExecutionSourceChunk;
-import org.folio.rest.jaxrs.model.ProfileSnapshotWrapper;
-import org.folio.rest.jaxrs.model.RawRecordsDto;
-import org.folio.rest.jaxrs.model.StatusDto;
+import org.folio.rest.jaxrs.model.*;
```

Comment:
Please, use single class import instead of wild card import

Reply:
@Maksat-Galymzhan  Thank you! It seems like after updating IntelliJIdea this setting was removed. Fixed.